.github/workflows/ci.yml

```
@@ -156,7 +169,7 @@ jobs:
         with:
           name: build-${{ matrix.target }}
           path: target/
-      - name: Integration Tests
+      - name: Integration Tests (without debian package)
```

Not sure why change of the name?

```suggestion
      - name: Integration Tests
```